Makefile

```diff
@@ -179,6 +179,10 @@ lint:
 unit:
 	hack/unit.sh ${GO_MOD_FLAGS}
 
+.PHONY: community-bundle
+community-bundle:
+	hack/community/generate.sh community-4.10 ARTIFACT_DIR # TODO: looking for ideas on how not to hardcode the version
```

Comment:
Possible way to do this: make generate.sh take in WMCO_VERSION (see top of file) instead of community branch
Then in generate.sh, create a mapping from wmco versions to ocp versions (wmco 6 to ocp 4.11, 5 to 4.11, etc). That way you’ll know if you’re given WMCO_VERSION=6.y.z to cut a release from branch community-4.11.